### PR TITLE
Doc: add e2e planning readme

### PR DESCRIPTION
**Pull request by JasonZhou404**

### CODE REVIEW

**Review by storypku — DISMISSED** [review]

File: modules/planning/README.md [hunk 1]
```diff
@@ -21,10 +21,12 @@ Apollo 6.0, which focuses on intellegence of autonomous driving, introduced two
 ## E2E Mode
    ![](images/e2e_mode.png)
 ### How to Enable 
+- Change the configuration "learning_mode" in apollo/modules/planning/conf/planning_config.pb.txt to be "E2E_TEST" if Apollo is run in simulation or "E2E" on real vehicle
+- Change the configuration "model_type" in apollo/modules/planning/conf/scenario/learning_model_sample_config.pb.txt to be either "CNN_LSTM" or "CNN" and adapt the following "cpu_model_file" and "gpu_model_file" file paths. "CNN_LSTM" is the preferred model for now.
 
 ### Model Inputs and Outputs
-
-### Parameters 
+- Model input is consisted of a birdview image centered by vehicle pose and vehicle current velocity.
```
Comment (jinghaomiao) [inline modules/planning/README.md:28]: consists

**Review by jinghaomiao — DISMISSED** [review]

File: modules/planning/README.md (hunk 1 above)
Comment (JasonZhou404) [reply modules/planning/README.md]: solved

File: modules/planning/README.md [hunk 2]
```diff
@@ -16,15 +16,19 @@ use cases as they were all treated as a single driving scenario.
 Apollo 5.5, which focuses on curb-to-curb autonomous driving on urban roads,
 introduced 2 new planning scenarios.
 
-Apollo 6.0, which focuses on intellegence of autonomous driving, introduced two new planning modes: E2E mode and Hybrid mode. 
+Apollo 6.0, which focuses on intellegence and data-driven development of autonomous driving, introduced two new planning modes: E2E mode and Hybrid mode, by which we demonstrate the new capability of dynamically nudginng a moving obstacle. These two modes also define a series of APIs where developers can generate thier own training data and brought in their own models.
```
Comment (storypku) [inline modules/planning/README.md:19]: `intellegent` as an adj for `development` ? 
`thier` should be `their`. `brought in` should be `bring in`.

File: modules/planning/README.md [hunk 3]
```diff
@@ -16,15 +16,19 @@ use cases as they were all treated as a single driving scenario.
...
+
+**Note** that the current development of E2E mode is in early stage where it is fined tuned more for the dynamic nudge scenario rather than comprehensive ones. So suboptimality is expected when running other scenarios in simulation. E2E mode and Hybrid mode are not tested on real vehicle yet. The work on data-driven planning is in still in progress and welcomes any collaboration with developer by any means including algorithms, models and data.
```
Comment (storypku) [inline modules/planning/README.md:21]: 1) **Note**: The current development ...

2) `fine-tuned`.  
3) `is in still in progress` => `is still in progress` .
4) `with developer` should be removed.

**Review by storypku — CHANGES_REQUESTED** [review]

**Review by jinghaomiao — APPROVED** [review]

**Pull request merged**